android/app/src/main/java/com/bounswe2020group3/paperlayer/RegisterFragment.kt

```diff
@@ -1,58 +1,20 @@
 package com.bounswe2020group3.paperlayer
 
+import android.app.Fragment
```

Comment:
Why using native Fragments ? AFAIK native Fragments are not recommended to use. Instead we should use either androidx fragment or support fragment.
https://www.andreasschrade.com/android-native-fragment-or-support-fragment-or-androidx-fragment

Reply:
These fragments were implemented by Ramazan and I did not change them. We can migrate to androidx and there wont be any problems since the keywords are identical for native and androidx library.

Reply:
All right, this is not a blocker for this pull request but we should be aware of it.